Source/dom/export.js

```diff
@@ -46,58 +46,142 @@ export const DEFAULT_EXPORT_OPTIONS = {
  *
  *
  * @param {dictionary} options Options indicating which sizes and formats to use, etc.
+ *
+ * @returns If an output path is not set, the data is returned
  */
 export function exportObject(object, options) {
-  const merged = { ...DEFAULT_EXPORT_OPTIONS, ...options }
+  if (!object) {
+    throw new Error('No object provided to export')
+  }
+  let { formats } = options
+  if (typeof formats === 'string') {
+    formats = [formats]
```

Comment:
So I think this should be `formats = formats.split(‘,’).map(format => format.trim())` because it is supposed to be a string of comma separated values. I’m fine with accepting arrays tho, it’s much cleaner. 

But the docs need to be changed and we still need to pass a string to the native method.

Reply:
Got it. I wasn't actually 100% sure if it was supposed to be an array or string. I think the docs say a "list".